src/main/java/org/junitpioneer/internal/PioneerPreconditions.java

```diff
@@ -60,10 +56,7 @@ public static String notBlank(String str, Supplier<String> messageSupplier) {
 	 * @return the supplied object
 	 */
 	public static <T> T notNull(T object, String message) {
-		if (object == null) {
-			throw new PreconditionViolationException(message);
```

Comment:
I think that this needs documentation update!

Reply:
Sorry, I don't follow. Why do you think this needs a documentation update?

Reply:
I have in mind that this Exception is explicity named in the documentation of at least one of our extension. But I'm not at my private PC so I can't search in source code if I'm right.

Reply:
> I have in mind that this Exception is explicity named in the documentation of at least one of our extension. But I'm not at my private PC so I can't search in source code if I'm right.

After searching the files, it seem's I'm wrong and mixing it up with

> Specifying more than one kind of parameter source (i.e.: annotating your test parameters and the test method itself) does not work and will throw an `ExtensionConfigurationException`.